packages/nitro-protocol/src/index.ts

```diff
@@ -22,4 +23,14 @@ export interface ChannelStorage {
 }
 
 // TODO: Export this with more thought to what is exposed by @statchannels/nitro-protocol
-export {createDepositTransaction, State, encodeConsensusData, Outcome, AllocationItem, Channel};
+export {
+  createDepositTransaction,
+  State,
+  encodeConsensusData,
+  Outcome,
+  AllocationItem,
+  Channel,
+  getChallengeRegisteredEvent,
+  isAllocationOutcome,
+  isGuaranteeOutcome,
+};
```

Comment:
It's worth grouping these exported symbols into something like (a) `utils` and (b) `types`